### PR TITLE
update pouchlinter to 0.2.4

### LINKED ISSUE #989: update golang 1.12.6 to 1.12.10
### Ⅰ. Issue Description
There is a CVE for Go and this vulnerability is fixed in 1.12.10. We should update it.
See: https://nvd.nist.gov/vuln/detail/CVE-2019-16276
### Some work need to do for updating go
- update ci

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### Ⅰ. Describe what this PR did
Update pouchlinter

### Ⅱ. Does this pull request fix one issue?

fixes #989

### CODE REVIEW

**Comment by allencloud** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,3 @@ script:
 
 # after_success:
 #   - bash <(curl -s https://codecov.io/bash) || echo "Failed to report code coverage!"
-
-cache:
```
Comment (allencloud) [inline .travis.yml]: Why to delete this, is this related to pouchlinter?

Comment (yeya24) [reply .travis.yml]: No. I think this config is for previous Java version. Do we still need that?

Comment (allencloud) [reply .travis.yml]: No, we don't.

**Pull request merged**